python/ray/_raylet.pyx

```diff
@@ -83,7 +83,8 @@ from ray.exceptions import (
     RayError,
     RayletError,
     RayTaskError,
-    ObjectStoreFullError
+    ObjectStoreFullError,
+    RayTimeoutError,
```

Comment:
I think we should just call this `TimeoutError` (it's already namespaced by being part of the `ray.exceptions` module).

Reply:
think that would overload the built-in `TimeoutError` for python3 (so would only work for 2.7)